### PR TITLE
Created JSClient, UpdatedCCBAPI, Benchmark

### PR DESCRIPTION
**Pull request by The-Cherry-Man**
Created three new classes, one for server implementation and two for speed tests. Also updated build.gradle for server

### CODE REVIEW

File: test/Benchmark.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import junit.framework.Assert;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigInteger;
+
+public class Benchmark {
+
```
Comment (jonathan7170) [inline test/Benchmark.java:7]: Needs Contructor

**Review by jonathan7170 — COMMENTED** [review]
Added contructor

File: test/Benchmark.java (hunk 1 above)
Comment (The-Cherry-Man) [reply test/Benchmark.java:7]: nice save

**Review by jonathan7170 — COMMENTED** [review]
Looks Good

**Review by jonathan7170 — APPROVED** [review]
looks good

**Pull request merged**